pages/index.js

```diff
@@ -49,32 +86,73 @@ export default function ShoppingItemsList({
             mode={mode}
           />
         )}
-
-        {mode !== "add" && (
-          <AddItemContainer>
-            <>
-              <p>...need anything else?</p>
+        {mode !== "add" ||
+          (!mode && (
+            <AddItemContainer>
+              <>
+                <p>...need anything else?</p>
+              </>
               <AddButton onClick={() => handleChangeMode("add")}>+</AddButton>
-            </>
-          </AddItemContainer>
+            </AddItemContainer>
+          ))}
+        {unpurchasedItems.length !== 0 && (
+          <div>
+            <StyledH2>
+              {/* {purchasedItems.length === 0 ? "We need " :  */}
+              There&#39;s {/* } */}
+              {unpurchasedItems.length} thing
+              {unpurchasedItems.length !== 1 ? "s" : ""}{" "}
+              {purchasedItems.length !== 0 ? "left to get" : "on the list"}:
```

Comment:
I understand that you want to differentiate between singular and plural here. Beyond that, I wouldn’t adjust the wording. In my opinion, it doesn’t add much value and only complicates the code unnecessarily. I suggest keeping it simple.

```suggestion
             You need {unpurchasedItems.length} thing
              {unpurchasedItems.length > 1 && "s"}:
```

Reply:
Thanks for your suggestion but my internal web designer is telling me that this is what they want. I'd rather keep it for now and postpone the decision to the Layout - US when I will discuss the issue again...